app/views/userTournament/chart.scala

```diff
@@ -14,26 +14,21 @@ object chart:
       path = "chart"
     ) {
       div(cls := "tournament-stats")(
-        boxTop(h1(userLink(u, withOnline = true), " tournament stats")),
+        boxTop(h1(trans.xTournamentStats.txt(userLink(u, withOnline = true)))),
```

Comment:
this is broken
![Q63ZuO5](https://github.com/lichess-org/lila/assets/140370/1562f79a-1998-4ea9-b1f6-cdec83ba4627)